Formula/gitversion.rb

```diff
@@ -0,0 +1,26 @@
+class Gitversion < Formula
+  desc "Easy semantic versioning for projects using Git"
+  homepage "https://github.com/GitTools/GitVersion"
+  url "https://www.nuget.org/api/v2/package/GitVersion.CommandLine/3.6.2"
+  version "3.6.2"
+  sha256 "db05ebacfb76ddca26180f0c70060fbb89800e434d3d55b524785d0da37dbac8"
+
+  bottle :unneeded
+
+  depends_on "mono"
+
+  def install
+    file = File.new("gitversion", "w", 0555)
+    file.write("#!/bin/sh\n")
+    file.write("#{HOMEBREW_PREFIX}/opt/mono/bin/mono #{libexec}/tools/GitVersion.exe \"$@\"\n")
+    file.flush
+    file.close
+
+    libexec.install Dir["*"]
+    bin.install_symlink "#{libexec}/gitversion"
+  end
+
+  test do
+    system "#{bin}/gitversion", "-?"
```

Comment:
I have a test now that sort of works, but I feel like it's brittle and may break in unexpected ways if run on environments that aren't exactly like my Mac:

```ruby
(testpath/"test.txt").write("test")
system "cd", testpath
system "git init"
system "git add test.txt"
system "git commit --author='Test <test@example.com>' -q -m 'Test'"
system bin/"gitversion -output json"
```

For some reason, the last line can't be written as such:

```ruby
system bin/"gitversion", "-output json"
```

If I do that, GitVersion fails with `Failed to parse arguments: -output json`. Well, well. The only thing remaining now is to assert the output from GitVersion and ensure that the setup of the Git repository doesn't break in other environments than mine.